packages/react-tooltip/src/components/TooltipProvider/renderTooltipProvider.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import { TooltipProviderState } from './TooltipProvider.types';
+import { getSlots } from '@fluentui/react-utilities';
+import { Tooltip } from '../../Tooltip';
+import { internal__TooltipContext } from '@fluentui/react-tooltip-trigger';
+
+/**
+ * Render the final JSX of TooltipProvider
+ * {@docCategory TooltipProvider}
+ */
+export const renderTooltipProvider = (state: TooltipProviderState) => {
+  const { slots, slotProps } = getSlots(state, []);
+
+  const { children, ...rootProps } = slotProps.root;
+
+  return (
+    <internal__TooltipContext.Provider
+      value={{
+        manager: state.manager,
+        portalRoot: state.portalRoot,
+        Tooltip,
```

Comment:
As `TooltipProvider` is injected into `FluentProvider` - `Tooltip` leaks into it. That results in bundle size growth as `Tooltip` will be included even if it is unused:

![image](https://user-images.githubusercontent.com/14183168/114699524-b53b2a80-9d20-11eb-96b7-e57e6fcc22a2.png)

Reply:
I've removed TooltipProvider from FluentProvider for now